.gitattributes

```diff
@@ -1 +1,4 @@
 /yb-voyager/src/utils/version.go export-subst
+
+# Exclude migration test files from language statistics
+migtests/tests/ linguist-generated
```

Comment:
https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github#:~:text=Use%20the%20linguist%2Dgenerated%20attribute%20to%20mark%20or%20unmark%20paths%20that%20you%20would%20like%20to%20be%20ignored%20for%20the%20repository%27s%20language%20statistics%20and%20hidden%20by%20default%20in%20diffs. 

`linguist-generated` seems to indicate to git that these are automatically generated, and as per doc above, they will be hidden by default in diffs. Doesn't seem like that would be a good idea... ?